package.json

```diff
@@ -22,7 +22,9 @@
     "compression-webpack-plugin": "~0.3.1",
     "css-loader": "^0.23.1",
     "dotenv": "^2.0.0",
-    "enzyme": "^2.4.1",
+    "enzyme": "^3.0.0",
+    "enzyme-adapter-react-15": "^1.0.0",
```

Comment:
I assume that once we upgrade react we just need to remove the adapter?

Reply:
Yeah actually we'll upgrade it to enzyme-adapter-react-15.